eslint/base/package.json

```diff
@@ -7,6 +7,7 @@
   "license": "MIT",
   "private": false,
   "dependencies": {
+    "@movable/eslint-plugin-no-wildcard-postmessage": "^1.0.0",
```

Comment:
I forked the original plugin, which had a dependency on ESLint 3, and published it again under our NPM namespace. In addition to some modernization, I set it up with out default set of lint rules and such, in case we end up maintaining our fork long-term.

I did PR my modernization efforts back to the original project, which can be found here:

https://github.com/mozfreddyb/eslint-plugin-no-wildcard-postmessage/pull/6

Since the plugin hasn't been maintained very actively, I don't know if that will actually land and itself be published to NPM. If it is someday, I may delete our fork and use the upstream package again, but the plugin is so simple that it might just be easier to keep our own fork that we have control over.